libbpf-tools/cvector.h

```diff
@@ -0,0 +1,198 @@
+
```

Comment:
You are right, I should have to.
I add commit hash number on the description.
And I'm thinking about to use other project by adding submodule. What do you think?

Reply:
I just copied only one file from each project.
So I agree with your opinion(getting file rather than using submodule).
And you said, I need to wait for maintainers to decide.